### DISCUSSION TITLE
websocket can not use

### DISCUSSION THREAD

**Discussion by coolmingwu** [tiangolo/uvicorn-gunicorn-fastapi-docker#322] [Questions] [answered]
After the websocket service is started, it becomes unavailable

**Comment by tiangolo** [comment, marked as answer]
It would be better to provide an example app to copy-paste and replicate the issue.

But anyway, now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image
